services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -733,7 +733,7 @@
                     },
                     {
                       "name": "option7",
-                      "label": "Other",
+                      "label": "Other, specify",
```

Comment:
Because this changed, the `"visibleOptionValue": "Other"` on line 745 also needs to be updated.